_includes/1.0/api/invoice.html

```diff
@@ -196,16 +189,26 @@ <h3 id="invoicejsonformat">JSON Format</h3>
 				    2: Partially Delivered<br />
 				    3: Delivered</td>
             </tr>
+            <tr>
+                <td>invoice_status</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>no</td>
+                <td>yes</td>
+                <td>The invoice status, among :<br />
+                    1: Not invoiced (default)<br />
+                    2: Invoiced<br />
+                    3: Cancelled</td>
```

Comment:
Actually, for invoices, it'll be only 2 or 3 (an Invoice is always at least Invoiced, not like a quote)

So it's more : 
```

<td>The invoice status, among :<br />
                    2: Invoiced (default)<br />
                    3: Cancelled</td>
```